in_toto/models/layout.py

```diff
@@ -120,6 +120,27 @@ def read(data):
 
     return Layout(**data)
 
+  def display(self):
+    """
+    <Purpose>
+      Represents the Layout class by returning its fields in a readable string
```

Comment:
Could you add a very condensed form of the rationale for why you choose to represent the layout exactly this way?

Reply:
Yup, done. Let me know if anything else is missing!